test/runtests.jl

```diff
@@ -1,6 +1,7 @@
 
 using Test, SHA, SparseArrays
 using SBML
+using SBML: Model, Reaction, Species
```

Comment:
I could also just change the test to say SBML.Model, but for now this is probably fine?

Reply:
yeah, this here is better.

Reply:
so this is resolved, right